src/main/java/com/example/loosecoupling/mapstruct/StudentMapper.java

```diff
@@ -0,0 +1,59 @@
+package com.example.loosecoupling.mapstruct;
+
+import com.example.loosecoupling.Entity.Student;
+import com.example.loosecoupling.dto.*;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface StudentMapper {
```

Comment:
Please clean up this mapper and remove the unused/unnecessary methods
Also, remove the unused DTOs.

Reply:
My StudnetDto & StudentCreationDto were exactly same, so that I removed StudentCreationDto, and used general StudentDto for request & response while creating new student.
Other Dtos are as it because every person type has different response view of student information.